src/test/resources/cz/startnet/utils/pgdiff/add_sequence_issue225_new.sql

```diff
@@ -0,0 +1,80 @@
+--
+-- PostgreSQL database dump
+--
+
+SET client_encoding = 'UTF8';
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+--
+-- Name: SCHEMA public; Type: COMMENT; Schema: -; Owner: postgres
+--
+
+COMMENT ON SCHEMA public IS 'Standard public schema';
+
+
+SET search_path = public, pg_catalog;
+
+--
+-- Name: testseq; Type: SEQUENCE; Schema: public; Owner: fordfrog
+--
+
+CREATE SEQUENCE alert_alert_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+CREATE SEQUENCE sequence_10
+    AS bigint
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public.alert_alert_id_seq OWNER TO fordfrog;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: testtable; Type: TABLE; Schema: public; Owner: fordfrog; Tablespace: 
+--
+
+CREATE TABLE testtable (
+    field1 integer,
+    field2 integer,
```

Comment:
This table diffing is not related to this patch, the test would be clearer if it's not included. Only the `CREATE/ALTER SEQUENCE` are needed.